services/s3/index.js

```diff
@@ -0,0 +1,39 @@
+'use strict';
+
+const {S3Client, GetObjectCommand} = require('@aws-sdk/client-s3');
+
+// Creates the S3 Client with a given profile
+// TO-DO use local stack instead of personal AWS
+const s3Client = new S3Client({
+    region: 'eu-west-2',
+    profile: 'tempus-broker-s3'
+});
+
+// validates that the S3 response is a JSON
+function validateS3Response(response) {
+    if (response.ContentType !== 'application/json') {
+        throw new Error(`${response.ContentType} content type is not supported`);
+    } else {
+        console.log('File retrieved from S3 is valid');
+    }
+}
+
+// Gets an object from a bucket based on key
+async function retrieveObjectFromBucket(bucket, objectKey) {
```

Comment:
The linter complains here when I add a try catch so I removed it

Reply:
Even just one wrapped around the whole function? I.e 
```
async function retrieveObjectFromBucket(bucket, objectKey) {
    try {
        const command = new GetObjectCommand({
            Bucket: bucket,
            Key: objectKey
        });

        const response = await s3Client.send(command);

        // validate response is valid file type
        validateS3Response(response);

        // convert response to JSON object
        const str = await response.Body.transformToString();
        return JSON.parse(str);
    } catch (err) {
        throw new Error(err);
    }
}
```
It might get caught upstream, but I think any `await` call to AWS should be in a try/catch.